src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/forge_of_souls.cpp

```diff
@@ -31,7 +31,7 @@ enum Yells
     SAY_SYLVANAS_OUTRO                          = 0
 };
 
-enum Gossip
+enum FOS_Gossip
```

Comment:
FOS_gossip maybe

but if is it the only gossip in forge of souls?? i doubt it, needs a more specific name if so

Reply:
I'd prefer without `_` but :shrug:

Reply:
If the only change you need to apply the pull request is this one, I have no problem doing it, but the pull request has been open for more than 19 days now and I'm really a bit discouraged about it.

Reply:
I know that I am doing it is a priority for the project, because it is a really insignificant change that very few people are going to appreciate, but for me it is really important because it gives me the courage to continue researching.

Reply:
FOS_ if I remember correctly, it was a recommendation from another member, that's why I gave it that name. I don't remember if I chose the name precisely.